samples/drivers/w1/scanner/src/main.c

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2022 Caspar Friedrich <c.s.w.friedrich@gmail.com>
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <zephyr/device.h>
+#include <zephyr/devicetree.h>
+#include <zephyr/drivers/w1.h>
+#include <zephyr/logging/log.h>
+#include <zephyr/zephyr.h>
+
+LOG_MODULE_REGISTER(main, CONFIG_LOG_DEFAULT_LEVEL);
+
+void w1_search_callback(struct w1_rom rom, void *user_data)
+{
+	LOG_INF("Device found; family: 0x%02x, serial: 0x%016llx", rom.family,
+		w1_rom_to_uint64(&rom));
+}
+
+void main(void)
+{
+	const struct device *dev = DEVICE_DT_GET(DT_NODELABEL(w1));
```

Comment:
Any reason not to use `DEVICE_DT_GET_ONE()`

Reply:
Nope, fixed.